sda-commons-server-key-mgmt/src/main/java/org/sdase/commons/keymgmt/model/ValueMappingModel.java

```diff
@@ -0,0 +1,44 @@
+package org.sdase.commons.keymgmt.model;
+
+import java.util.*;
+
+@SuppressWarnings("unused")
+public class ValueMappingModel {
+
+  private final Map<String, String> apiToImplBidirectional = new HashMap<>();
+  private final Map<String, String> implToApiBidirectional = new HashMap<>();
+
+  private final Map<String, String> implToApi = new HashMap<>();
+  private final Map<String, String> apiToImpl = new HashMap<>();
+
+  public ValueMappingModel setApiToImplBidirectional(List<ValueMapping> apiToImplBidirectional) {
+    apiToImplBidirectional.forEach(
+        vm -> {
+          this.apiToImplBidirectional.put(vm.getApi(), vm.getImpl());
+          this.implToApiBidirectional.put(vm.getImpl(), vm.getApi());
+        });
+    return this;
+  }
+
+  public ValueMappingModel setImplToApi(List<ValueMapping> implToApi) {
+    implToApi.forEach(vm -> this.implToApi.put(vm.getImpl(), vm.getApi()));
```

Comment:
Do we need to clear "implToApi" first? Otherwise I would prefer "add..." as method name.

Reply:
cleared...